src/main/resources/l10n/JabRef_en.properties

```diff
@@ -456,6 +456,12 @@ Importing=Importing
 
 Importing\ in\ unknown\ format=Importing in unknown format
 
+Import\ CSS=Import CSS
+
+Import\ custom\ CSS\ file=Import custom CSS file
+
+Theme\ change\ to\ a\ custom\ theme.=Theme change to a custom theme.
```

Comment:
Please move the translation down to line 2129 - to group the theme translations together.